lib/components/DateTime/Calendar.tsx

```diff
@@ -316,132 +239,165 @@ export class Calendar extends React.Component<CalendarProps, Partial<CalendarSta
             start.date += 1;
         }
 
-        const content = rows.map((row, rowIndex) => {
-            let inner = row.map((col, colIndex) => {
-                const onClick = (event) => {
-                    this.onClick(col);
-                    event.preventDefault();
-                };
+        return rows;
+    }, [displayDate, localTimezone]);
 
-                const date = col.date;
-                const colMonth = col.month;
-                const key = `${colMonth}-${date}`;
-                const ariaLabel = new Date(`${curYear}-${colMonth + 1}-${date}`).toLocaleDateString(this.props.locale, { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' });
+    React.useEffect(() => {
+        if (nextFocusRow.current != null && nextFocusCol.current != null) {
+            const nextFocus = containerRef.current.querySelectorAll(`[data-row="${nextFocusRow.current}"][data-col="${nextFocusCol.current}"]`)[0] as HTMLElement;
+            if (nextFocus != null) {
+                nextFocus.focus();
+            }
+            nextFocusRow.current = undefined;
+            nextFocusCol.current = undefined;
+        }
+    });
+
+    const content = rows.map((row, rowIndex) => {
+        let inner = row.map((col, colIndex) => {
+            const handleClick = (event: React.SyntheticEvent) => {
+                onClick(col);
+                event.preventDefault();
+            };
+
+            const date = col.date;
+            const colMonth = col.month;
+            const key = `${colMonth}-${date}`;
+            const ariaLabel = new Date(`${curYear}-${colMonth + 1}-${date}`).toLocaleDateString(props.locale, { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' });
+
+            /** Grayed out day from another month */
+            if (colMonth !== curMonth) {
+                return (
+                    <Attr.button
+                        type='button'
+                        aria-label={ariaLabel}
+                        data-row={rowIndex}
+                        data-col={colIndex}
+                        onKeyDown={onKeyDown}
+                        className={colClassName}
+                        onClick={handleClick}
+                        key={key}
+                        attr={props.attr?.dateButton}
```

Comment:
nit: do `const { dateButton, dateRow... } = props.attr ?? {}` so we don't do the null check on attr multiple times?